bin/container-build.sh

```diff
@@ -1,34 +1,50 @@
 #!/bin/bash
-
 set -ex
 
-if [[ $# != 1 ]]; then
-  echo "Wrong number of arguments: $#. Usage example: container-build.sh <branch or tag>"
+while getopts "t:r:h" opt; do
+  case $opt in
+    t) BUILD_TYPE=$OPTARG ;;
+    r) REF=$OPTARG ;;
+    h) echo "Usage: $0 -t BUILD_TYPE -r REF [-h]"; exit 1
+  esac
+done
+
+if [ "$BUILD_TYPE" != "nightly" ] && [ "$BUILD_TYPE" != "release" ]; then
+  echo "Build type (-t) is required, must be either 'nightly' or 'release'"
+  exit 1
+fi
+
+if [ -z "$REF" ]; then
+  echo "ref (-r) is required"
   exit 1
 fi
 
-BRANCH=${1%%-*}
+BRANCH=${REF%%-*}
 PODS_SOURCE_DIR="/build/manageiq-pods-${BRANCH}"
 MANAGEIQ_SOURCE_DIR="/build/manageiq-${BRANCH}"
 
-if [ "${1}" = "master" ]; then
+if [ "${REF}" = "master" ]; then
   tag="latest"
-elif [ "${1}" != "${BRANCH}" ]; then # tag build
-  tag="${1}"
+elif [ "${REF}" != "${BRANCH}" ]; then # tag build
+  tag="${REF}"
 else # branch build
   tag="latest-${BRANCH}"
 fi
 
 rm -rf ${PODS_SOURCE_DIR}
-git clone -b ${1} https://github.com/ManageIQ/manageiq-pods ${PODS_SOURCE_DIR}
+git clone -b ${REF} https://github.com/ManageIQ/manageiq-pods ${PODS_SOURCE_DIR}
```

Comment:
Can this be a `--depth=1` for a performance boost (or do we need the history)?

Reply:
Added `depth` option.